referenceframe/model.go

```diff
@@ -8,12 +8,16 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 	pb "go.viam.com/api/component/arm/v1"
 
 	"go.viam.com/rdk/spatialmath"
 )
 
+// unsupportedFileType is returned if we try to build a model from an inproper extension.
+const unsupportedFileType = `only files with .json and .urdf file extensions are supported`
```

Comment:
`errUnsupportedFileType = errors.New(`

Reply:
To add to this, use quotes for strings, not backticks